requirements/requirements.in

```diff
@@ -9,6 +9,10 @@ django-compressor>=2.3
 django-model-utils
 django-debug-toolbar
 SALTISE_user_feedback
+pandas
```

Comment:
If these are production dependencies, you will need to add them to setup.cfg under `install_requires`

Reply:
I see. Silly question, but I thought that was handled by `requirements.in`. What's the difference?